doc-content/enterprise-only/integration/entando-con.adoc

```diff
@@ -2,6 +2,8 @@
 
 = Entando Digital Experience Platform (DXP) and App Builder
 
-Entando Digital Experience Platform (DXP) is a low-code, component-based platform for building modern web, mobile, and IoT applications. The Entando App Builder is the application that provides features to help you configure and customize, with or without programming skills, any kind of modern web application built with the Entando Platform. The Entando App Builder is a web-based administration console that you can use to build and configure your web application, manage your page data and activities, and integrate with other application tools, such as {PRODUCT}.
+Entando Digital Experience Platform (DXP) is a low-code, component-based platform for building modern web, mobile, and IoT applications. The App Builder is an application that provides features to help you configure and customize, with or without programming skills, any kind of modern web application built with the App Builder Platform. The App Builder is a web-based administration console that you can use to build and configure your web application, manage your page data and activities, and integrate with other application tools, such as {PRODUCT}.
```

Comment:
"the App Builder Platform"  Make sure to remove "Platform" here too, so it's "built with App Builder", with the change you make per my previous comment.

Reply:
Done